### ISSUE TITLE
A11y - OU: Context menu button lacking focus

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#5532]
The context menu in the OU listing page has an issue with lacking to display focus.

!Screen Shot 2024-11-19 at 11.42.51 AM.png|width=2162,height=572,alt="Screen Shot 2024-11-19 at 11.42.51 AM.png"!

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-39692**



[created from NEXT-39692, comment 513942]

**Comment by shopwareBot** [comment]
[public] 

 > FixVersion Change: **6.7.0.0**

 This issue will be / was solved in Shopware version ***6.7.0.0***.



[created from NEXT-39692, comment 527908]
